test/src/test/java/org/corfudb/infrastructure/TestServerRouter.java

```diff
@@ -4,16 +4,18 @@
 import lombok.Getter;
 import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
+import org.corfudb.protocols.service.CorfuProtocolMessage;
 import org.corfudb.protocols.wireprotocol.CorfuMsg;
 import org.corfudb.protocols.wireprotocol.CorfuMsgType;
```

Comment:
The testing path doesnt actually test the encode/decode code paths. Since we are writing new code, I suggest we start writing proper test cases using mocks and abandon the current TestRouter, its slow, erroneous and has bad coverage.